content/post/kenobi-boot2-root-writeup.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "Kenobi Boot2Root Write-Up"
+date: 2020-11-14T13:43:28-04:00
+author: "Nate Catelli"
+tags: ["ctf", "boot2root", "hacking", "writeup", "tryhackme"]
+description: "A boot2root writeup of the Kenobi host from TryHackMe"
+type: "post"
+draft: false
+---
+
+## Introduction:
+The Kenobi boot2root was a ton of fun because it required multiple pivots to learn enough to leak a key. On my first pass i ended up overlooking the NFS shares which really impressed the importance of carefully reviewing the scans.
+
+## Environment
+The attack take place on a flat network consisting of my attack host, which is a freshly-booted Kali Linux livecd, and the the target host, a freshly-booted Linux VM that I knew contained 2 flags to capture. Nothing else was known about the host prior to the attack.
```

Comment:
The attack took place on a flat network consisting of the attack host, a freshly-booted Kali Linuxcd, and the target host, which I knew held 2 flags to capture.

Nothing else about the host was known prior to the attack